src/components/try-hcp-callout/components/try-hcp-callout-compact/index.tsx

```diff
@@ -0,0 +1,29 @@
+import CardLink from 'components/card-link'
+import {
+	ProductIconHeading,
+	Description,
+	StandaloneLinkContents,
+} from 'components/try-hcp-callout/components'
+import { TryHcpCalloutProps } from 'components/try-hcp-callout/types'
+import s from './try-hcp-callout-compact.module.css'
+
+/**
+ * Renders a compact HCP themed callout card, without a mock UI visual.
+ * Intended for use with "Try HCP" callouts.
+ */
+export function TryHcpCalloutCompact({
+	productSlug,
+	heading,
+	description,
+	ctaText,
+	ctaUrl,
+}: TryHcpCalloutProps) {
+	return (
+		<CardLink className={s.root} ariaLabel={ctaText} href={ctaUrl}>
+			<div className={s.background} />
+			<ProductIconHeading productSlug={productSlug} headingText={heading} />
+			<Description description={description} />
+			<StandaloneLinkContents text={ctaText} />
+		</CardLink>
+	)
+}
```

Comment:
☝️ I'm on the fence!

IMO adding the `normal | compact` prop has the benefit of drying up the code a bit... But I think it also has the disadvantage of setting up a path of conditional rendering (eg for the "image" part). I figured the duplication of things like `.backgroundBase` is worth the tradeoff - especially since `.background` classes, while similar, use different `background-image` assets and `background-position`. And I feel like `CardLink` does a great job of handling the semantic concerns around `ariaLabel`, and the smaller "atomic" elements like `ProductIconHeading` also do a bit of that handling as well. 

So personally I lean slightly towards keeping the components separate - especially since we have another "variant" in use on the home page (not rolled in this PR, but maybe something we'd want to approach later as Kendall mentioned).

Admittedly it does feel a lil awkward in some ways (eg the `StandaloneLinkContents` interface & `:hover` work in the thread above)... hopefully that'll improve a bit as the approach matures (eg with subsequent work in 🎟 [Assess abstracting atomic "callout" components](https://app.asana.com/0/1202097197789424/1203396252179033/f), see also [ExpandableArrow](https://react-components.vercel.app/components/expandablearrow) as Alex linked).

All that being said, definitely open to refactoring here if there are strong opinions on taking the `normal|compact` & conditional rendering approach 🙇

Reply:
I lean towards having a small bit of duplication as opposed to conditional rendering in this case as well. I like the composed variants and makes the code more bite-sized / easier to iterate on IMO.

Reply:
Oh, I don't think we'd need to use conditional rendering! I see the prop being used to apply certain classes, and then the CSS would handle the style differences between the two types.

On the note of semantics and markup order what I meant by divergence is that because these are two separate components, they can easily diverge in what they pass to `CardLink`'s `ariaLabel`, the order in which they render the `children` passed to `CardLink`, and the ability for their CSS properties to change in the nearly identical classes. If these are things we want to remain consistent, then this feels like too much flexibility and too much to manually maintain.

Reply:
@ashleemboyer I suppose that's true, we could hide the `image` part in CSS rather than conditionally rendering it. To me it feels pretty similar.

The risk of unintentional divergence feels relatively low in my opinion - I think our `CardLink` component does a great job of handling the order for the link elements.

Rather than combining `TryHcpCallout` and `TryHcpCalloutCompact` here, maybe there's a case building for like `CardLinkWithBackground` that has `.backgroundBase` stuff built in? (Since that's something we use in a few other places, like the callout slot cards on the home page). Maybe that would help reduce some of the duplicative code here, while keeping the composition-y approach - feels like getting backgrounds to work as expected in `CardLink` will definitely come up again 👍

I'd be very on board to explore abstracting out the `CardLink` background work as part of 🎟 [Assess abstracting atomic "callout" components](https://app.asana.com/0/1202097197789424/1203396252179033/f). I'll add a lil breadcrumb note to that task - let me know what y'all think! 🙇

Reply:
> The risk of unintentional divergence for ariaLabel and critical element ordering feels relatively low in my opinion - I think our CardLink component does a great job of handling the order for the link elements.

To clarify -- I'm not referring to what `CardLink` does under the hood. I'm referring to:

- The actual values of `ariaLabel` passed to `CardLink` by the `TryHcpCallout` and `TryHcpCalloutCompact`. They both currently pass `ctaText` as `ariaLabel`. We don't want that value to diverge between the two components because it would cause them to have different screen reader announcements & voice control commands, and it would likely break the voice control experience.
- The order in which `TryHcpCallout` and `TryHcpCalloutCompact` arrange the elements they pass to `CardLink`

Reply:
I think we're referring to the same!

Since both components pass the `ctaText` prop directly, have a shared type, and aren't doing additional duplicate logic to generate the `ariaLabel` (say, like we might do in `TutorialsCard`), I personally think the risk of unintentional divergence is low. Similarly, personally I think the order of elements seems unlikely to be changed unintentionally.

It may be that we introduce new forms of the Try HCP callout that maybe have different compositions of the same elements, or it may be that we change one of the existing compositions (say, we decide to remove the `description` from the "compact" composition). In those cases we can intentionally have the different compositions diverge, and I don't think we'd introduce any new issues 🤔

Reply:
To me it feels like an unnecessary risk to already vulnerable users and unnecessary maintenance for us when we have the ability to prevent both and doing so would be low effort and a one-time, initial sort of effort too. 🤷🏻 But we can disagree about it and keep moving forward. Primarily wanted to make sure we were referring to the same thing.

Reply:
☝️ Great point! Would definitely be on board with pursuing any and all efforts to reduce the risk & reduce maintenance - if you want to describe the fix you're suggesting on 🎟 [Assess abstracting atomic "callout" components](https://app.asana.com/0/1202097197789424/1203396252179033/f) (would that be the `normal | compact` prop?), would definitely be open to taking another look at it as part of follow-up work ☺️ I feel like cause of the non-trivial, but not overly complex scope here, it might be a good kind of case study for the tradeoffs of the various approaches we can come up with 💯